### PR TITLE
Added Footer to show collapsible Cart Summary and Checkout button for ScanStore page

### PR DESCRIPTION
**Pull request by devYaoYH**
- Removed default .body 8px margin
  - Replaced with 8px padding for container in microapp environment
  - Needed to do "100vh" fixed grid container height in `ScanStore` page
- Added `CartSummary` component to display cart totals
- Moved price computation into temporary stand-in `src/priceLibrary.ts`
  - Need to compute in cents as `int` to avoid floating-point errors
- Re-layout `ScanStore` page with fixed `100vh` grid with `Header | ScrollBody | Footer` style
- Added collapsible price total detail footer
- Moved checkout button to be fixed at bottom and show total pricing when footer is collapsed

Demo:
ScanAndGo GPay (3)

### CODE REVIEW

**Comment by devYaoYH** [comment]
Raised issue #179 . Possibly we would need to migrate many of our 'default' strings into some `strings.json` file for fast localization - language options?

File: client/src/pages/ScanStore/ScanStore.tsx [hunk 1]
```diff
@@ -130,48 +139,91 @@ function ScanStore() {
...
+          <Grid item style={{ paddingBottom: theme.spacing(2) }}>
+            <Button
+              fullWidth={true}
+              variant="contained"
+              color="primary"
+              onClick={makePayment}
+            >
+              {showFooter
+                ? "Checkout"
+                : `Checkout: $${cartTotal.toFixed(PRICE_FRACTION_DIGITS)}`}
```
Comment (devYaoYH) [inline client/src/pages/ScanStore/ScanStore.tsx:222]: Raised issue #179 . Possibly we would need to migrate many of our 'default' strings into some `strings.json` file for fast localization - language options?

Comment (cszqwe) [reply client/src/pages/ScanStore/ScanStore.tsx:222]: That is a good question. 
We can treat this as a good to have feature.

**Comment by devYaoYH** [comment]
Test are fixed with #182, will wait for PR #182 approval and merge into here prior to pushing into master

**Pull request merged**